### ISSUE TITLE
Attribute size not allowed on element input at this point.

### ISSUE THREAD

**Issue by neuhaus** [validator/validator#2001]
### Spec URL

https://html.spec.whatwg.org/#the-input-element

### Test case

```html
<input id="test" type="number" size="4">
```

**Comment by neuhaus** [comment]
The Nu Html Checker is complaining that the size attribute isn't allowed here. It claims:

**[size](https://html.spec.whatwg.org/#attr-input-size)**: when type is [text](https://html.spec.whatwg.org/#text-(type=text)-state-and-search-state-(type=search)), [search](https://html.spec.whatwg.org/#text-(type=text)-state-and-search-state-(type=search)), [url](https://html.spec.whatwg.org/#url-state-(type=url)), [tel](https://html.spec.whatwg.org/#telephone-state-(type=tel)), [email](https://html.spec.whatwg.org/#e-mail-state-(type=email)), or [password](https://html.spec.whatwg.org/#password-state-(type=password))

I don't see this restriction in the spec, has it been altered?
In my case, type is `number`.

**Comment by nschonni** [comment]
The table in the linked spec only has "yes" for those types, and is empty for number

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/2001#issuecomment-3694528328
